### ISSUE TITLE
comparison of Fixnum with nil failed (ArgumentError)

### ISSUE THREAD

**Issue by dpetzel** [sensu/sensu#1007]
I've been getting this on occasion when restarting the service. Normally I second attempt at starting does the trick, but today, I'm actually stuck in a state where it won't start at all. 

```
service sensu-server start
Starting sensu-server                                      [FAILED]
```

From the log (snipped):
```
{"timestamp":"2015-07-23T20:52:07.618669+0000","level":"warn","message":"reconnecting to redis"}
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-0.18.1/lib/sensu/server/process.rb:282:in `>=': comparison of Fixnum with nil failed (ArgumentError)
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-0.18.1/lib/sensu/server/process.rb:282:in `check_flapping?'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-0.18.1/lib/sensu/server/process.rb:312:in `block in update_event_registry'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/em-redis-unified-0.6.0/lib/em-redis/redis_protocol.rb:458:in `call'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/em-redis-unified-0.6.0/lib/em-redis/redis_protocol.rb:458:in `dispatch_response'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/em-redis-unified-0.6.0/lib/em-redis/redis_protocol.rb:412:in `process_cmd'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/em-redis-unified-0.6.0/lib/em-redis/redis_protocol.rb:386:in `receive_data'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/eventmachine-1.0.3/lib/eventmachine.rb:187:in `run_machine'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/eventmachine-1.0.3/lib/eventmachine.rb:187:in `run'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-0.18.1/lib/sensu/server/process.rb:23:in `run'
	from /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-0.18.1/bin/sensu-server:10:in `<top (required)>'
	from /opt/sensu/bin/sensu-server:23:in `load'
	from /opt/sensu/bin/sensu-server:23:in `<main>'
```

Is there something I need to clear/delete to get things moving again? Looking through the code it seems related to flap detection, but I'm simply trying to get the service started.

**Comment by portertech** [comment]
This was a bug that was fixed in 0.19 https://github.com/sensu/sensu/blob/master/CHANGELOG.md#0190---2015-06-01 I recommend upgrading to 0.20 :+1:

**Comment by dpetzel** [comment]
@portertech awesome, thank you, sorry I missed it had already been fixed. Will certainly look at upgrading.
